buildchain/buildchain/versions.py

```diff
@@ -350,7 +350,7 @@ def deb_full_name(self) -> str:
         PackageVersion(
             name='calico-cni-plugin',
             version=CALICO_VERSION,
-            release='0'
+            release='1'
```

Comment:
You need to fix `sos-report` as well (it's build ID should be 1 not 0)

The only reason why it was working for `sos` is because WE build the package (and thus were passing 0 to the docker) so passing the buggy ID at two places made things work by accident (but it's still wrong).

(dont' forget to update the commit message accordingly when you amend the commit)

Reply:
Done